test-common/src/main/kotlin/GenerelleTestData.kt

```diff
@@ -23,6 +23,9 @@ import java.time.temporal.TemporalUnit
 val fixedClock: Clock =
     Clock.fixed(1.januar(2021).atTime(1, 2, 3, 456789000).toInstant(ZoneOffset.UTC), ZoneOffset.UTC)
 
+/** Fixed UTC clock at 2021-02-01T01:02:03.456789000Z */
+val enUkeEtterFixedClock: Clock = fixedClock.plus(7, ChronoUnit.DAYS)
```

Comment:
```suggestion
val enUkeEtterFixedClock: Clock = fixedClock.plus(1, ChronoUnit.WEEKS)
```
https://docs.oracle.com/javase/8/docs/api/java/time/temporal/ChronoUnit.html#WEEKS

Reply:
https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#plus-long-java.time.temporal.TemporalUnit-